### DISCUSSION TITLE
Praca domowa #2

### DISCUSSION THREAD

**Discussion by macborowy** [cloudstateu/ic-1-2022#4] [Announcements]
Witajcie :wave:,

w ramach przekazania nam odpowiedzi do pracy domowej #2 prosimy Was o udostępnienie linku do aplikacji Frontend. Jeśli Ingress będzie poprawnie skonfigurowany będziemy w stanie sprawdzić czy API posiada endpoint `/health` i `/ready` 😈  

Odpowiedzi udostępnijcie w komentarzu 👇

**Comment by PawelTo** [comment]
Cześć,
Serdeczna prośba, czy moglibyście wcommitować rozwiązania nad którymi pracowaliśmy w poprzedni czwartek :)
